documentation/modules/about-cold-warm-migration.adoc

```diff
@@ -47,3 +47,51 @@ The precopy stage runs until the cutover stage is started manually or is schedul
 The VMs are shut down during the cutover stage and the remaining data is migrated. Data stored in RAM is not migrated.
 
 You can start the cutover stage manually by using the {project-short} console or you can schedule a cutover time in the `Migration` manifest.
+
+[id="warm-migration-versus-cold-migration"_{context}]
+== Cold migration vs. warm migration: Which is right for your migration?
+
+Both warm migration and cold migration have advantages and limitations. In general, warm migrations can take longer than cold migrations, but warm migrations involve less downtime than cold migrations.
+
+The table that follows describes the advantages and limitations of each type of migration. It assumes that you have installed Red Hat Enterprise Linux (RHEL) 9 on the {ocp} platform on which you installed {project-short}.
+
+[cols="1,1,1",options="header"]
+.Advantages and limitations of cold and warm migrations
+
+|===
+| |Cold migration |Warm migration
+
+|*Duration*
+| Each block is copied only once, making the duration of the migration shorter.
+| A block might be copied in more than one snapshot. {project-short} ensures that only the most recent snapshot is used, but eliminating duplicate snapshots adds time to the duration of the migration.
+
+|*Fail fast*
+| Each block is converted to be compatible with {ocp-short} and, if the conversion is successful, the block is transferred. If a block cannot be converted, the migration fails immediately.
+| All blocks are transferred before any are converted. This means that it takes longer for a warm migration to fail than a cold migration, because a cold migration fails before all blocks are transferred.
```

Comment:
@mnecas thoughts on this?

Reply:
```
Each block is converted to be compatible with {ocp-short} and, if the conversion is successful, the block is transferred. If a block cannot be converted, the migration fails immediately.
```
We do not convert each block. The warm migration in the MTV perspective means creating a snapshot and transferring the snapshot. Once there is a cutover we create the last snapshot, which consists only of the changes between the previous snapshot and the last one. For that we use and **require** Changed Block Tracking (CBT) on the VM and VM disk to be enabled. Once the last snapshot is transferred we do commit the layers and run the guest conversion on the resulting image. After the guest conversion is finished we do create the VM and start it.
Something like this:
![image](https://github.com/user-attachments/assets/ac8ebd90-1ca8-48a4-be66-3427cfb88058)

Reply:
I would not say each block. It fails faster because it could fail before the disk transfer. Please take a look at:
https://access.redhat.com/articles/1353223

Unlike warm migrations we are using the virt-v2v directly for both disk transfer and guest conversion

Reply:
@anarnold97 As you can see above, I'm going to need to mention the requirement for CBT.  Martin requested I link to this article https://knowledge.broadcom.com/external/article/320557/changed-block-tracking-cbt-on-virtual-ma.html , but I see that it includes a procedure. What do you think about a sentence like "Warm migration requires Changed Block Tracking (CBT), which is described in VMware documentation" and leave it at that, even though that  forces the user to do the research?

Reply:
We don't need the article, we need to say that we **require** the feature and say how to enable it on the VM, which is described in the article solution. ie add advanced parameters:
```
ctkEnabled = TRUE
scsi0:0.ctkEnabled = TRUE
....
scsiX:X.ctkEnabled = TRUE
```